mailbox/tools/indexer/src/main/java/org/apache/mailbox/tools/indexer/ErrorRecoveryIndexationTaskDTO.java

```diff
@@ -92,14 +98,17 @@ public String getMailboxId() {
     }
 
     private final String type;
-    private final List<ReindexingFailureDTO> previousFailures;
+    private final List<ReindexingFailureDTO> messageFailures;
+    private final  Optional<List<String>> mailboxFailures;
     private final Optional<RunningOptionsDTO> runningOptions;
 
     private ErrorRecoveryIndexationTaskDTO(@JsonProperty("type") String type,
-                                           @JsonProperty("previousFailures") List<ReindexingFailureDTO> previousFailures,
+                                           @JsonProperty("previousFailures") List<ReindexingFailureDTO> messageFailures,
```

Comment:
Should we call this property `previousMessageFailures`?

Reply:
we can't change the @JsonProperty, only the param name?